pkg/settings/setting.go

```diff
@@ -95,7 +95,7 @@ var (
 	IgnoreNodeName                    = NewSetting("ignore-node-name", "") // nodes to ignore when syncing v1.node to v3.node
 	NoDefaultAdmin                    = NewSetting("no-default-admin", "")
 	RestrictedDefaultAdmin            = NewSetting("restricted-default-admin", "false") // When bootstrapping the admin for the first time, give them the global role restricted-admin
-	EKSUpstreamRefreshCron            = NewSetting("eks-refresh-cron", "*/5 * * * *")
+	ClusterUpstreamRefreshCron        = NewSetting("eks-refresh-cron", "*/5 * * * *")
```

Comment:
As discussed in meeting we'll have EKS and GKE and AKS, but take out cron name and cron format. There will be conversion method from cron to int representing seconds in the controller.

Reply:
So what is the upgrade path for a user that already used the old setting? Changing the name creates a new setting CR.

Reply:
Yes we will be migrating old setting in cron format to new setting in seconds format, then deleting the old setting.